internal/cmd/kafka/command_link_create.go

```diff
@@ -167,36 +193,124 @@ func (c *linkCommand) create(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+var sensitiveLinkConfigs = map[string]bool{
```

Comment:
How likely is it that we add more in the future? Will we ever need to retroactively mark configs as sensitive?

Reply:
It's a good question. It is of course possible. I think it's unlikely though. @rajinisivaram what do you think?

Reply:
Chatted with @rajinisivaram offline. We'll move to an allow list to be on the safe side.